layers/terrain-mesh.js

```diff
@@ -328,4 +358,21 @@ export class TerrainMesh extends BufferedMesh {
       this.gpuTasks.delete(key);
     }
   }
+
+  setPackage(pkg) {
+    const {textures} = pkg;
+    const diffuseAtlas = new CanvasTextureAtlas(textures[DIFFUSE_MAP], THREE.sRGBEncoding, SUB_TEXTURE_SIZE);

```

Comment:
This looks like it's doing a bake, which should have already happened when we made the package. Setting the baked package is probably not the time to bake.

Reply:
This should be moved to package creation time.